build.gradle

```diff
@@ -23,61 +28,65 @@ repositories {
     maven { url "https://mvnrepository.com/artifact/com.google.inject/guice" }
     maven { url "https://mvnrepository.com/artifact/org.zeromq/jeromq" }
     maven { url "https://frcmaven.wpi.edu/artifactory/release/"}
-    maven { url "http://devsite.ctr-electronics.com/maven/release/"}
-    maven { url "http://www.kauailabs.com/maven2" }
-    maven { url "http://www.revrobotics.com/content/sw/max/sdk/maven/"}
+    maven { url "https://devsite.ctr-electronics.com/maven/release/"}
+    maven { url "https://www.kauailabs.com/maven2" }
+    maven { url "https://www.revrobotics.com/content/sw/max/sdk/maven/"}
     maven { url uri ("../SeriouslyCommonLib/lib/")}
 }
 
 configurations {
     nativeBundle
 }
 
-List<String> wpilibDesktopJni(String platform) {
```

Comment:
wpi.wpilibVersion doesn't exist anymore, I just pulled all this out and it seemed to be okay but maybe something will burn us later